components/member-profile.jsx

```diff
@@ -0,0 +1,33 @@
+var React = require('react');
+var slug = require(`slug`);
+
+slug.defaults.mode = `rfc3986`;
+
+var MemberProfile = React.createClass({
+  propTypes: {
+    name: React.PropTypes.string.isRequired,
+    title: React.PropTypes.string,
+    twitter: React.PropTypes.string,
+    pic: React.PropTypes.string,
+    bio: React.PropTypes.object
+  },
+  render: function() {
+    return (
+      <div className="row m-y-3 member-profile" id={slug(this.props.name)}>
+        <div className="col-sm-3 center-vertical">
+          <img className="circle" src={this.props.pic || `/assets/images/team/placeholder.jpg` }/>
+        </div>
+        <div className="col-sm-9">
+          <h3 className="m-b-0 name"><a href={`#`+slug(this.props.name)}>{this.props.name}</a></h3>
```

Comment:
oh this is def missing `id`. In the past we got requests from people that anchors were hard to find so they requested the anchored element to show up as link.